### PR TITLE
Add section on using Coverity Scan

### PR DESCRIPTION
**Pull request by martyone**

### CODE REVIEW

**Comment by martyone** [comment]
Depends on JB#61781, JB#61824 and JB#61829. Do not merge until those are released.

**Comment by saukko** [comment]
LGTM

**Review by saukko — APPROVED** [review]

File: Develop/Platform/Testing_Advice/README.md [hunk 1]
```diff
@@ -192,4 +192,53 @@ Above combines three different endurance data sets and generates *index.html* an
...
+
+Static analysis with Coverity Scan is essentially a two step process from its user point of view. First the Coverity Scan Build Tool is used to capture the source code in an intermediate format, accompanied with information on how it is built. The captured data are then submitted for the actual analysis by the Coverity Scan service.
+
+The most convenient way to accomplish the first step is by using the `cov-build` tool, available from the `bin` subdirectory of the downloaded archive. Compared to the [alternative ways](https://sig-product-docs.synopsys.com/bundle/coverity-docs/page/coverity-analysis/topics/build_capture_for_compiled_languages.html), it works by intercepting calls to the compiler as it is invoked by the build system during a regular build process.
+
+There is one issue with the `cov-build` tool though. Is has the bad habit of polluting `stdout` with its informal messages, which prevents its use deeper in the build stack. Work this around by creating a wrapper script for `cov-build` and use this wrapper in next steps instead.
```
Comment (rainemak) [inline Develop/Platform/Testing_Advice/README.md:207]:
```suggestion
There is one issue with the `cov-build` tool though. It has the bad habit of polluting `stdout` with its informal messages, which prevents its use deeper in the build stack. Work this around by creating a wrapper script for `cov-build` and use this wrapper in next steps instead.
```

**Review by rainemak — APPROVED** [review]
Small typo spotted. Approving as well.

**Comment by martyone** [comment]
rebased

**Pull request merged**